presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -335,6 +351,22 @@ public PlanNode visitRemoteSource(RemoteSourceNode node, RewriteContext<Void> co
         }
 
         @Override
+        public PlanNode visitMergeRemoteSource(MergeRemoteSourceNode node, RewriteContext<Void> context)
```

Comment:
Do you ever have a MergeRemoteSourceNode when UnaliasSymbolExpressions is called? (e.g., is there a RemoteSource visitor here -- if not, you don't need a MergeRemoteSourceNode)